tests/console/commands/test_lock.py

```diff
@@ -28,6 +42,19 @@ def poetry_with_old_lockfile(project_factory, fixture_dir, source_dir):
     )
 
 
+@pytest.mark.skipif(
+    sys.platform == "win32", reason="does not work on windows under ci environments"
```

Comment:
What's happening under windows? Do you know why?

Reply:
Hey, wha'd'ya know?  Totally unnecessary anymore :)

Reply:
Perfect :+1: Could you please add a test for a non outdated lock file as well?

Reply:
I've added a new test and refactored the testing a bit.  I keep getting failures on the FreeBSD/Py36 tests while installing dependencies, though.  All of the other Py36 testing is passing, so I can only figure that something is broken on that test runner.

Reply:
Ok, today tests are all passing, including the new test :)